src/main/java/com/practice/ts/PracticeTwo/controllers/HomeController.java

```diff
@@ -0,0 +1,42 @@
+package com.practice.ts.PracticeTwo.controllers;
+
+import com.practice.ts.PracticeTwo.models.CommentDto;
+import com.practice.ts.PracticeTwo.models.StoryItemDto;
+import com.practice.ts.PracticeTwo.repositories.CommentRepo;
+import com.practice.ts.PracticeTwo.repositories.StoryItemRepo;
+import com.practice.ts.PracticeTwo.services.HackerNewsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class HomeController {
```

Comment:
Best practice is to have your controller only interact with services. Controllers really should never know about the repositories. In this class you're wiring in your repos but really you need another service that handles interacting with hacker news and storing your entities in the background.